test/benchmarks/sequencer/common/transactions/transactions.go

```diff
@@ -29,24 +29,35 @@ func SendAndWait(
 	log.Debugf("Sending %d txs ...", nTxs)
 	startingNonce := auth.Nonce.Uint64()
 	maxNonce := uint64(nTxs) + startingNonce
+	initialPendingCount, err := countByStatusFunc(params.Ctx, pool.TxStatusPending)
+	if err != nil {
+		panic(err)
+	}
 
 	for nonce := startingNonce; nonce < maxNonce; nonce++ {
-		err := txSenderFunc(client, auth.GasPrice, nonce, auth, erc20SC)
+		err = txSenderFunc(client, auth.GasPrice, nonce, auth, erc20SC)
 		if err != nil {
+			for err != nil && err.Error() == "nonce intrinsic error" {
```

Comment:
This was caused by a bug in the RPC that was already fixed. Anyway nice to have.